src/tests/playwright/pages/dsp-graph-evaluation.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <script type="module" defer>
+      import { evaluateGraph } from './util/audio-comparer.js';
+      import { test, evaluate } from './util/audit.js';
+
+      // My Render Length (seconds)
+      // noinspection JSConstantReassignment,JSValidateTypes
+      test((async () => {
+        const length = 1;
+        // My DSP Graph (function)
+        const myGraph = ctx => {
+          const osc = new OscillatorNode(ctx);
+          osc.type = 'sawtooth';
+
+          const gain = new GainNode(ctx);
+          gain.gain.value = 2.2;
+
+          const biq = new BiquadFilterNode(ctx);
+          biq.type = 'bandpass';
+          biq.frequency.value = 1000;
+          biq.Q.value = 10;
+
+          osc.connect(gain).connect(biq).connect(ctx.destination);
+          osc.start();
+          osc.stop(ctx.currentTime + length);
+        }
+
+        const score = await evaluateGraph(myGraph, length);
+        return {score};
+      })());
+
+      evaluate(async () => {
+        const score = (await window.test).score;
```

Comment:
Is this the recommended pattern from Playwright? Relying on `window.test` seems out-of-date.

Reply:
If this pattern is unavoidable, please make the name more distinctive. (e.g. `webAudioTest` or something similar)